CONTRIBUTING.md

```diff
@@ -31,8 +31,8 @@ facilitates social contribution, easy testing and peer review.
 To contribute a patch, the workflow is as follows:
 
   1. Fork Repository
-  2. Create topic branch
-  3. Commit patches
+  1. Create topic branch
+  1. Commit patches
```

Comment:
@storopoli nit: is this index right ?

Reply:
there are also some other ones in the other .md files, but commenting here to not spam the PR

Reply:
It's good practice to use 1. indexing in `.md` files (see why [here](https://www.seancdavis.com/posts/pro-tip-use-1-for-every-ordered-list-item-when-writing-markdown/)).

Under the hood everything becomes

```html
<ol>
  <li>Create topic branch</li>
  <li>Commit patches</li>
</ol>
```

Then diffing and maintaining is a breeze.
Because if you move or remove item 2, you are screwed.
You need to rename all the other ones.
And the diff will suck.
With this you are fine.
Move, move, remove, `while true { list.shuffle() }`, you are good to go.

Reply:
Cool, I didn't know that ;)
Thanks for the explanation and resource #TIL